libtransmission/resume.h

```diff
@@ -44,6 +44,7 @@ auto inline constexpr Filenames = fields_t{ 1 << 20 };
 auto inline constexpr Name = fields_t{ 1 << 21 };
 auto inline constexpr Labels = fields_t{ 1 << 22 };
 auto inline constexpr Group = fields_t{ 1 << 23 };
+auto inline constexpr QueueState = fields_t{ 1 << 25 };
```

Comment:
Where is `1 << 24`?

Reply:
#6746

Reply:
These two PRs seem closely related; does 6746 need to land first?